ee/modules/110-istio/hooks/migration_remove_kiali_crds.go

```diff
@@ -1,23 +0,0 @@
-/*
```

Comment:
Exclude this hook please, we are refactoring it in https://github.com/deckhouse/deckhouse/pull/2410.

Reply:
Done